Sources/SwifterSwift/UIKit/UIScrollViewExtensions.swift

```diff
@@ -32,6 +32,107 @@ public extension UIScrollView {
+    }
+
+    /// SwifterSwift: Scroll to the bottom-most content offset.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollToBottom(animated: Bool = true) {
+        setContentOffset(CGPoint(x: contentOffset.x, y: max(0, contentSize.height - bounds.height) + contentInset.bottom), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll to the right-most content offset.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollToRight(animated: Bool = true) {
+        setContentOffset(CGPoint(x: max(0, contentSize.width - bounds.width) + contentInset.right, y: contentOffset.y), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll up one page of the scroll view.
+    /// If `isPagingEnabled` is `true`, the previous page location is used.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollUp(animated: Bool = true) {
+        let minY = -contentInset.top
+        var y = max(minY, contentOffset.y - bounds.height)
+        #if !os(tvOS)
+        if isPagingEnabled,
+            bounds.height != 0 {
+            let page = max(0, ((y + contentInset.top) / bounds.height).rounded(.down))
+            y = max(minY, page * bounds.height - contentInset.top)
+        }
+        #endif
+        setContentOffset(CGPoint(x: contentOffset.x, y: y), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll left one page of the scroll view.
+    /// If `isPagingEnabled` is `true`, the previous page location is used.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollLeft(animated: Bool = true) {
+        let minX = -contentInset.left
+        var x = max(minX, contentOffset.x - bounds.width)
+        #if !os(tvOS)
+        if isPagingEnabled,
+            bounds.width != 0 {
+            let page = ((x + contentInset.left) / bounds.width).rounded(.down)
+            x = max(minX, page * bounds.width - contentInset.left)
+        }
+        #endif
+        setContentOffset(CGPoint(x: x, y: contentOffset.y), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll down one page of the scroll view.
+    /// If `isPagingEnabled` is `true`, the next page location is used.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollDown(animated: Bool = true) {
+        let maxY = max(0, contentSize.height - bounds.height) + contentInset.bottom
+        var y = min(maxY, contentOffset.y + bounds.height)
+        #if !os(tvOS)
+        if isPagingEnabled,
```

Comment:
Could you add just a comment explaining why this is necessary? It may help someone reading this in the future :)

Reply:
There's a comment already in the documentation of the function that it will scroll to the next page in that direction.

Reply:
Oops wrong line, I meant the tvOS check :)

Reply:
`isPagingEnabled` doesn't exist in `tvOS`